usbSerialForAndroid/src/main/java/com/hoho/android/usbserial/driver/UsbId.java

```diff
@@ -41,6 +41,10 @@ public final class UsbId {
     public static final int QINHENG_CH340 = 0x7523;
     public static final int QINHENG_CH341A = 0x5523;
 
+    public static final int VENDOR_UNISOC = 0x1782;
+    public static final int FIBO_L610 = 0x4D10;
```

Comment:
please change FIBO to FIBOCOM, the full company name. There are significantly less google hits for the incomplete name